README.md

```diff
@@ -0,0 +1,5 @@
+# hacktober
```

Comment:
Could you change the contents of the README file to say its about learning git only

Thanks etc can be done via email